app/services/participants/withdraw/validate_and_change_state.rb

```diff
@@ -16,6 +16,11 @@ def perform_action!
         ActiveRecord::Base.transaction do
           ParticipantProfileState.create!(participant_profile: user_profile, state: ParticipantProfileState.states[:withdrawn], reason: reason)
           user_profile.training_status_withdrawn!
+
+          unless user_profile.npq?
+            induction_coordinator = user_profile.school&.induction_coordinator_profiles&.first
```

Comment:
school is required for non-npq profiles, we should be fine with dropping both `&` here.